src/list_cpu_features.c

```
@@ -70,39 +97,84 @@ DEFINE_PRINT_FLAGS(GetMipsFeaturesEnumValue, GetMipsFeaturesEnumName,
                    MipsFeatures, MIPS_LAST_)
 #endif
 
+//
+// Program entry
+//
 int main(int argc, char** argv) {
+
+  char *optionJsonFile = NULL;
+  int idx = 0;
+
+  for (idx = 1; idx < argc; ++idx) {
+
+    if ((strcmp(argv[idx], "-h") == 0) || (strcmp(argv[idx], "--h") == 0)) {
+        showUsage(argc, argv);
+        exit(EXIT_FAILURE);
```

Not sure getting the help should return an error, I would return SUCCESS aka 0.